cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `namespace` attribute in the `include` function matches the namespaces defined in the respective `urls.py` files of the 'cinema' and 'user' apps. This is important for Django's URL resolution to work correctly.